### DISCUSSION TITLE
[Feature suggestion] Get notifications only in "packet mode"

### DISCUSSION THREAD

**Discussion by roigreenberg** [zepp-health/zeppos-forum#7] [Developer Feedback]
In addition to notifying only when the phone screen is closed, can you please add an option to notify only when the phone is "in the packet"? (meaning, when something is covering the proximity sensor)
I think it can be very useful, and improve the battery life.

The reason for that is simple.
I want to get notifications when the phone is not available for me.
If it is my hand, or sitting in front of me, there is no reason to send notifications.

The behavior, in that case, should be:
Phone unlocked -> no notifications
Phone locked on the table -> no notifications
Phone locked in my packet -> send notifications

The use case is very simple. When I work on my computer, I put the phone in front of me, but I have WhatsApp web and telegram web, so I don't need my phone for that, but when I text, the watch is buzzing like crazy.

**Comment by Xuemuyang** [comment]
Thank you very much for your feedback!

I will send this suggestion back to the relevant colleagues.

**Comment by raylewisdesign** [comment]
This is a good idea!
But the upsetting fact is the watch does not have access to tell if the phone is unlocked, not to mention whether it is in your pockets or not :(

**Reply by roigreenberg** [reply]
You don't need the watch to know, you need the app on your phone, and it can know it easily (get notification only when the phone is locked is already exists)

**Reply by htoooth** [reply]
sound good
